m4/pbs_default_scheduler.m4

```diff
@@ -37,12 +37,7 @@
 
 AC_DEFUN([PBS_AC_DEFAULT_SCHEDULER],
 [
-  AC_MSG_CHECKING([for Default scheduler name])
```

Comment:
I'm not sure I agree.  The purpose of pbs_share.h is for sharing defines and other data between daemons.  This data is hardcoded and does not change at configure time.  IMHO, using autogen will make this more complicated than it needs to be.